src/workerd/api/node/crypto-keys.c++

```diff
@@ -71,7 +72,23 @@ CryptoImpl::AsymmetricKeyDetails CryptoImpl::getAsymmetricKeyDetail(
 }
 
 kj::StringPtr CryptoImpl::getAsymmetricKeyType(jsg::Lock& js, jsg::Ref<CryptoKey> key) {
-  KJ_UNIMPLEMENTED("not implemented");
+  static std::map<kj::StringPtr, kj::StringPtr> mapping{
```

Comment:
I think this requires a lock on every invocation, can we make it constexpr top-level?

Reply:
I modeled this after several other places where we are following the same pattern. I think I'll keep this like it is for now and if we think it's better with constexpr top-level then we can change all of them.